tests/integration/test_charm.py

```diff
@@ -49,13 +56,56 @@ async def test_group_membership(self, ops_test: OpsTest):
         url = await get_unit_url(
             ops_test, application=APP_NAME, unit=0, port=6080
         )
-        url = f"{url}/service/xusers/groupusers"
-        response = requests.get(
-            url, headers=HEADERS, auth=RANGER_AUTH, timeout=20
+        membership = await get_memberships(ops_test, url)
+
+        assert membership == ("commercial-systems", 8)
+
+    async def test_simulate_crash(self, ops_test: OpsTest):
+        """Simulate the crash of the Ranger charm.
+
+        Args:
+            ops_test: PyTest object.
+        """
+        # Destroy charm
+        await ops_test.model.applications[APP_NAME].destroy(force=True)
```

Comment:
We're usually discouraged from using the `--force` flag when removing applications as it might mean there are underlying issues when the application is removed that should be resolved without the need to force delete the application. As you are only relating to the PostgreSQL charm, I suspect the `force=True` might not even be necessary.

Reply:
Makes sense, removed.